setup.py

```diff
@@ -30,4 +30,6 @@
           'console_scripts': [
               'ims-nparser = imsnpars.cli:main',
           ],
-      })
+      },
+      python_requires=">=3.7"
```

Comment:
Muss das wirklich 3.7 sein? Funktioniert auch 3.6?

Reply:
Ich glaube das hat sich aus dem original Repository ergeben und wurde übernommen. Wenn du Tests für ältere Versionen durchlaufen lässt, können wir das gerne abändern.

Reply:
Naja. Bisher steht im Fork und im Orginal Repo die Version 3.7 nur in der Readme.md